### DISCUSSION TITLE
Documentation popup in non-GUI Emacs

### DISCUSSION THREAD

**Discussion by computersarecool** [company-mode/company-mode#1178] [Q&A] [answered]
Just to confirm as mentioned in #138  there is no support for the documentation pop-ups while using a non window-system Emacs in company mode, correct?

Is  it correct that Company-quick help does not allow pop-ups in Emacs without a window system?

**Comment by dgutov** [comment, marked as answer]
It is correct.

And [company-quickhelp](https://github.com/expez/company-quickhelp) is a separate package, not part of `company-mode`.

It has been recently (2020) transferred to the same organization, though.
